packages/node_modules/@cerebral/http/src/HttpProviderError.js

```diff
@@ -1,11 +1,11 @@
 export default class HttpProviderError extends Error {
-  constructor(status, headers, body, message = null, isAborted = false) {
+  constructor(status, headers, result, message = null, isAborted = false) {
     super()
     this.name = 'HttpProviderError'
     this.message = message
     this.status = status
     this.headers = headers
-    this.body = body
+    this.result = result
```

Comment:
Not sure I understand the reason to use 'result' instead of 'body'. "Result" is a very generic term that does not rely add any meaning. I think the only places this parameter is used is with "xhr.responseText" so why not name it "responseText" instead ? This is a well documented Wep API name.